package.json

```diff
@@ -36,17 +36,18 @@
   },
   "devDependencies": {
     "babel": "^5.8.23",
-    "babel-eslint": "^4.1.3",
-    "babel-loader": "^5.3.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
     "eslint-plugin-react": "^3.5.1",
     "html-loader": "^0.4.3",
     "html-webpack-plugin": "^2.10.0",
-    "prop-types": "~15.5.8",
-    "react": "~15.5.4",
-    "react-dom": "~15.5.4",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-hot-loader": "^1.3.0",
-    "webpack": "^1.10.1",
-    "webpack-dev-server": "^1.10.1",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^4.6.0",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
same.. you'll need to fix https://github.com/gozego/react-infinite-scroll-component/blob/master/server.js and https://github.com/gozego/react-infinite-scroll-component/blob/master/webpack.config.js

Reply:
@techgaun do you want to help me with those? updating these was the only way to get rid of the vulnerability in the package "open", but I lack the knowledge to figure out what else needs to change for these in webpack.config.js and server.js.

Reply:
I removed the updates to webpack in the latest commit. Should not need the updates to config files now.